### DISCUSSION TITLE
connecting to two mqtt brokers

### DISCUSSION THREAD

**Discussion by HaoSDEmoN** [Koenkk/zigbee2mqtt#11694] [Show and tell]
Hi!
Is it possible to connect zigbee2mqtt to two brokers?
I didn't find any information on the website and forum.
If possible, tell me how to implement a connection to two brokers.

**Comment by Koenkk** [comment]
Not possible

**Comment by mpfl** [comment]
It might be possible to bridge the two MQTT brokers. In this configuration, one broker logs into the other and duplicates all the messages sent on a particular topic (all topics is possible as well).

But without know which broker you're using, I cannot provide any more information.
